src/test/scala/beam/agentsim/agents/ElectricVehicleChargingBehaviorTest.scala

```diff
@@ -412,34 +444,34 @@ class ElectricVehicleChargingBehaviorTest
     val closestTAZsCAV = List("12")
```

Comment:
The test `"Ride Hail Electric vehicles" should "pick chargers choosing smaller DrivingTimeCost."` is sometimes getting stuck on CI, this should be fixed before merging. I'll investigate.